pymc_marketing/mmm/risk_assessment.py

```diff
@@ -0,0 +1,461 @@
+        Risk aversion parameter (0 = low risk aversion, 1 = high risk aversion).
+
+    Returns
+    -------
+    ObjectiveFunction
+        A function that calculates the adjusted Value at Risk score given samples and budgets.
+
+    Raises
+    ------
+    ValueError
+        If the risk aversion parameter is not between 0 and 1.
+        If confidence_level is not between 0 and 1.
+    """
+
+    def _adjusted_value_at_risk_score(
+        samples: np.ndarray, budgets: np.ndarray
+    ) -> float:
+        if not 0 <= risk_aversion <= 1:
+            raise ValueError("Risk aversion parameter must be between 0 and 1.")
+        if not 0 < confidence_level < 1:
+            raise ValueError("Confidence level must be between 0 and 1.")
+        # Calculate the empirical VaR directly from the samples
+        var = np.percentile(samples, (1 - confidence_level) * 100)
+        mean = np.mean(samples)
+        return (1 - risk_aversion) * mean + risk_aversion * var
+
+    return _adjusted_value_at_risk_score
+
+
+def portfolio_entropy(samples: np.ndarray, budgets: np.ndarray) -> float:
+    R"""
+    Calculate the entropy of a portfolio's asset weights to assess diversification.
+
+    Portfolio entropy, derived from Shannon entropy in information theory, quantifies
+    the dispersion of asset weights within a portfolio. A higher entropy value indicates
+    a more diversified portfolio, as investments are more evenly distributed across assets.
+    Conversely, a lower entropy suggests concentration in fewer assets, implying higher risk.
+
+    The entropy is calculated using the formula:
+
+    .. math::
+        E = -\sum_{i=1}^{n} w_i \cdot \log(w_i)
+
+    where :math:`w_i` represents the weight of asset \( i \) in the portfolio.
+
+    Parameters
+    ----------
+    assets : np.ndarray
+        1D array representing the investment amounts in each asset.
+
+    Returns
+    -------
+    float
+        Portfolio entropy value.
+
+    References
+    ----------
+    - Bera, A. K., & Park, S. Y. (2008). Optimal Portfolio Diversification using the Maximum Entropy Principle.
+    - Pola, G. (2013). On entropy and portfolio diversification. *Journal of Asset Management*, 14(4), 228-238.
+    """
+    weights = budgets / np.sum(budgets)
+    entropy = -np.sum(weights * np.log(weights))
+    return entropy
+
+
+def diversification_ratio(samples: np.ndarray, budgets: np.ndarray) -> float:
+    R"""
+    Calculate the Diversification Ratio of a portfolio to evaluate risk distribution.
+
+    The Diversification Ratio measures the effectiveness of diversification by comparing
+    the weighted average volatility of individual assets to the overall portfolio volatility.
+    A higher ratio indicates better diversification, as it reflects lower correlations among
+    assets, leading to reduced portfolio risk.
+
+    The Diversification Ratio is calculated as:
+
+    .. math::
+        DR = \frac{\\sum_{i=1}^{n} w_i \\cdot \\sigma_i}{\\sigma_p}
```

Comment:
This is not rendering well in the documentation. I think it has to dow ith the double backslash and R string. \\sum -> \sum, etc 

Similar to #1085